### PR TITLE
qa|RMD-3418 add js build scripts for each environment

### PR DESCRIPTION
**Pull request by betterin30days**
[RMD-3418](https://rubiconmd.atlassian.net/browse/RMD-3418)

## Changes
Added an npm build script for each env

## Questions
- this renames `build` to `build-prod`. I just want to confirm that there is not some automated build process somewhere that is relying on this convention. I believe everything here is manual.

### CODE REVIEW

**Comment by iovis** [comment]
> this renames build to build-prod. I just want to confirm that there is not some automated build process somewhere that is relying on this convention. I believe everything here is manual.

You're good

File: package.json [hunk 1]
```diff
@@ -20,7 +20,10 @@
   },
   "scripts": {
     "test": "non",
-    "build": "gulp build --env='prod'",
+    "build": "gulp build --env='local'",
+    "build-qa": "gulp build --env='qa'",
+    "build-demo": "gulp build --env='dev'",
```
Comment (iovis) [inline package.json]: These don't do anything unless you make it do something in the Gulpfile: https://github.com/rubiconmd/rubiconmd-js/blob/ce46f4e9aac7eb2506ff661de4d13af5f6f13bc2/gulpfile.js#L36-L46

**Comment by iovis** [comment]
I'm unsure right now with the context of this PR, neither the description nor the JIRA shed light on it. This didn't change any behavior as per my comment above, so I don't know what the end goal for this was.

I'll need a bit more context to review

**Comment by betterin30days** [comment]
@iovis You're definitely correct. I was looking at the [env file](https://github.com/rubiconmd/rubiconmd-js/blob/master/app/env/RubiconMDEnv.js) and thinking I needed these changes. The env in gulp just minifies it or not.

I should have put more context. The intent was for us to be able to build and deploy to qa, demo, or prod. But thanks for catching this as this does not do that.

I'll make changes so I just have `build` and `build-prod` to either minify or not. We can deploy the results of `build` to qa and `build-prod` to demo and prod.

**Pull request merged**